ts/features/services/screens/ServiceDetailsScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { ScrollView, StyleSheet } from "react-native";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { ServiceId } from "../../../../definitions/backend/ServiceId";
+import { IOStackNavigationRouteProps } from "../../../navigation/params/AppParamsList";
+import { ServicesParamsList } from "../navigation/params";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { serviceByIdSelector } from "../../../store/reducers/entities/services/servicesById";
+import { loadServiceDetail } from "../../../store/actions/services";
+import { ServicePublic } from "../../../../definitions/backend/ServicePublic";
+import { useHeaderSecondLevel } from "../../../hooks/useHeaderSecondLevel";
+
+export type ServiceDetailsScreenNavigationParams = Readonly<{
+  serviceId: ServiceId;
+  // if true the service should be activated automatically
+  // as soon as the screen is shown (used for custom activation
+  // flows like PN)
+  activate?: boolean;
+}>;
+
+type ServiceDetailsScreenProps = IOStackNavigationRouteProps<
+  ServicesParamsList,
+  "SERVICE_DETAIL"
```

Comment:
```suggestion
   typeof SERVICES_ROUTES.SERVICE_DETAIL
```

Reply:
imo it should be the same _(in the codebase usually we use the "string" way)_

Reply:
If you have to change it, you change only in one place. I mean the "string" name. In this case you need to change in two place. You loose the TS strong typing feature. BTW, it's was only a suggestion, I prefer the DRY way.